corehq/apps/analytics/static/analytix/js/appcues.js

```diff
@@ -0,0 +1,49 @@
+/* global Appcues, Array, window */
+
+/**
+ * Instantiates the AppCues analytics and customer support messaging platform.
+ */
+hqDefine('analytix/js/appcues', [
+    'underscore',
+    'analytix/js/initial',
+    'analytix/js/logging',
+    'analytix/js/utils',
+], function (
+    _,
+    initialAnalytics,
+    logging,
+    utils
+) {
+    'use strict';
+    var _get = initialAnalytics.getFn('appcues'),
+        _logger = logging.getLoggerForApi('Appcues'),
```

Comment:
`getLoggerForApi` needs to be called again inside of the document ready handler - which is terrible, but that's where we are right now. See https://github.com/dimagi/commcare-hq/pull/18899 for explanation.